src/modules/ui/SignInButt.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@mui/material/Button';
+
+const SignInButt = () => {

```

Comment:
This component isn't reusable. Also, spell out the full component. Consider renaming to "SignInButton".

Reply:
Or better yet, to abide by the principles of DRY rename to or `RedButton` then add customisation props As this button might be used heavily throughout the app.